### ISSUE TITLE
UI mock up for country page / country sub-site

### ISSUE THREAD

**Issue by anderspeders** [NRGI/resourcecontracts.org#177]
We will need to have a page for each country. This could essentially be identical significant overlaps with the country subsite UI. 

This should gives the overview of: 
- how many contracts are there
- signed in different years and 
- categorized under different resources
- geocoordinates and georeference (as these will be incorporated into the metadata)

**Comment by anjesh** [comment]
See the mockup https://projects.invisionapp.com/share/VZ3SHI63A#/screens/94423784 
Currently we don't have the examples of georeferences, could be taken care of if there are some examples.

**Comment by jedm** [comment]
Similar to [UX challenge](https://github.com/NRGI/resourcecontracts.org/issues/186) on homepage, the [country page layout](https://projects.invisionapp.com/share/VZ3SHI63A#/screens/94423784) should emphasize a fewer number of visual and functional priorities, and allow others to recede to smaller options or 1 click away. For instance:

* Since user has selected a country of interest, full facet list on left is not necessary. It should be reachable, but not crowd in to create too many equal options on Country mainpage.

* A prominent, good-looking way to click into most recent contract, or immediately begin filtering by resource, or start an “only-this-country” search, should take more visual priority than automated summaries of yearly totals or by-resource totals. These options to narrow further should be presented like functions to click (and totals should be available but relegated to smaller size, supplemental data).

Layout of this page (and the single contract page) needs to emphasize functional options more so it guides user into most useful actions, while still surfacing key attributes and metadata.

Other functions that should probably have “real estate” on the Country page (no particular order):

* Link to single-country API
* Option for notification when new contract for this Country is added
* Automated list of most recent blog entries related to that country
* Visible function/prompt for comparing one country’s contract or terms to another country’s
* Toolbox for sharing (for instance, so I can filter down to Country X/Resource Y/Year Range Z-CC and then share that selected filter-state via email, etc.).

Basically, demote automated totals in favor of contracts (titles, icons) and visible, user-friendly tools for browsing, narrowing, filtering, sharing, comparing. 

https://projects.invisionapp.com/share/VZ3SHI63A#/screens/94423784

**Comment by byndcivilization** [comment]
@anjesh can we use a jquery drawer to show and hide the facet list?

**Comment by byndcivilization** [comment]
something like this for instance
http://git.blivesta.com/drawer/

**Comment by anjesh** [comment]
We need to now work on this while working on the subsite docker. Right now the country site follows the structure of the RC and OLC but the front page needs new look/design.

**Comment by anjesh** [comment]
Closing this, as we have already setup and deployed multiple country sites.
